opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -366,16 +366,16 @@ mod tests {
         assert_eq!(data_point.value, 50, "Unexpected data point value");
     }
 
-    #[ignore = "https://github.com/open-telemetry/opentelemetry-rust/issues/1065"]
     #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
     async fn counter_aggregation_overflow_delta() {
         counter_aggregation_overflow_helper(Temporality::Delta);
+        counter_aggregation_overflow_helper_custom_limit(Temporality::Delta);
     }
 
-    #[ignore = "https://github.com/open-telemetry/opentelemetry-rust/issues/1065"]
     #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
     async fn counter_aggregation_overflow_cumulative() {
         counter_aggregation_overflow_helper(Temporality::Cumulative);
+        counter_aggregation_overflow_helper_custom_limit(Temporality::Delta);
```

Comment:
```suggestion
        counter_aggregation_overflow_helper_custom_limit(Temporality::Cumulative);
```

Reply:
It looks like you have changed both of the methods to use `Delta` instead of `Cumulative`.